src/tests/RunApp.cs

```diff
@@ -21,15 +21,14 @@ public class AppTest
         [Fact]
         public async Task RunApp()
         {
-            string[] args = new string[] { "--kvname", "heliumtest-kv", "--authtype", "CLI"  };
-
-            await Helium.App.Main(null);
-            await Helium.App.Main(new string[] { "--help" });
-            await Helium.App.Main(new string[] { "--kvname", " " });
+            string[] args = new string[] { "-k", "heliumtest-kv", "-a", "CLI", "-d"  };
 
             Task<int> t = Helium.App.Main(args);
 
-            await Task.Delay(10000);
+            args = new string[] { "-k", "heliumtest-kv", "-a", "CLI" };
+            t = Helium.App.Main(args);
+
+            await Task.Delay(5000);
 
 
             using HttpClient client = new HttpClient { BaseAddress = new System.Uri("http://localhost:4120") };
```

Comment:
Should we assert the result values in this test? at least the response codes particularly with the invalid parameter cases?

Reply:
the response codes and body read will throw an exception, so no need to assert